CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.5 (Apr 05, 2022)
+
+IMPROVEMENTS:
```

Comment:
Wondering if this PR is patch version change or a minor version change. Looks like not just a documentation update like mentioned in the description.

Reply:
it is more than documentation change.

**request retry mechanism introduced on specific error conditions**